vagrant/provisioning/roles/httpd/templates/ldap-auth-restricted-linuxadmins-helper.script2

```diff
@@ -0,0 +1,14 @@
+AuthType Basic
+AuthName "Restricted applications"
```

Comment:
Name this "Authentication Required". The AuthName will be displayed on the password prompt from the browser.